styles/trinket/catppuccin.user.css

```diff
@@ -0,0 +1,1455 @@
+
+      img {
+        box-shadow: 0 0 0 10px @accent-color !important;
+      }
+
+      a.social {
+        background-color: @accent-color !important;
+
+        i::before {
+          color: @base !important;
+        }
+      }
+    }
+
+    #outputTabs {
+      background-color: @crust !important;
+      border-bottom-color: @surface0 !important;
+
+      .active {
+        background-color: @mantle !important;
+      }
+
+      #instructionsTab {
+        border-left-color: @surface0 !important;
+      }
+    }
+
+    #recent-trinkets .item a {
+      background-color: @mantle !important;
+      border-color: @surface0 !important;
+      box-shadow: none !important;
+    }
+
+    #content-wrapper {
+      border-color: @surface0 !important;
+    }
+
+    #wrapper {
+      border-color: @surface0 !important;
+
+      .trinket-content {
+        border-top-color: @surface0 !important;
+      }
+
+      .trinket-wrapper .trinket-content {
+        background-color: @base !important;
+      }
+
+      .trinket-label {
+        color: @subtext0 !important;
+      }
+
+      &,
+      .trinket-content-wrapper > :not(.codeOutput, #content-overlay) {
+        background-color: @base;
+      }
+
+      .codeOutput {
+        background-color: initial !important;
+      }
+
+      .left-menu-toggle {
+        border-right-color: @surface0 !important;
+      }
+
+      .right-menu-toggle {
+        border-left-color: @surface0 !important;
+      }
+
+      .icon-bar .item.split {
+        &.button,
+        &.faux-button {
+          span {
+            border-left-color: @surface0 !important;
+          }
+
+          span::after {
+            border-top-color: @text !important;
+          }
+        }
+      }
+    }
+
+    #content-overlay {
+      background-color: rgba(@base, 0.2) !important;
+    }
+
+    #books-index .book-img {
+      outline-color: @surface0 !important;
+
+      &:hover {
+        outline-color: @accent-color !important;
+      }
+    }
+
+    #svg-logo .letters {
+      fill: @text !important;
+    }
+
+    #svg-logo .yellow {
+      fill: @yellow !important;
+    }
+
+    #svg-logo .blue {
+      fill: @blue !important;
+    }
+
+    #svg-logo .green {
+      fill: @green !important;
+    }
+
+    #svg-logo .white {
+      fill: @base !important;
+    }
+
+    #blue-logo-circle {
+      // There normally isn't an animation applied to this
+      // element because the circle blends in to the background.
+      opacity: 0;
+      animation: RtL 1s 0.75s ease 1 forwards !important;
+    }
+
+    #hero {
+      background-color: @mantle !important;
+    }
+
+    #graphic {
+      // This is the element that is the code output for Python graphics (and probably more)
+      // If styled with catppuccin, it may make previous code projects unviewable because of
+      // color contrast. This can still be overwritten (at least in python) by using
+      // `turtle.Screen().bgcolor("some color")` from the `turtle` module.
```

Comment:
```suggestion
      /* This is the element for graphics output. If styled with Catppuccin, it may make graphics unviewable due to contrast issues. */
```